### PR TITLE
Fixes #24587 - Modules Streams associated to a CVV

### PR DESCRIPTION
**Pull request by parthaa**
UI bindings to show module streams associated to a content view version
Piggy backs off https://github.com/Katello/katello/pull/7698 and https://github.com/Katello/katello/pull/7688

Look at the last commit for the changes here
cvv

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-module-streams.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      <tr bst-table-row ng-repeat="module in table.rows">
+        <td bst-table-cell>{{ module.name }}</td>
+        <td bst-table-cell>{{ module.stream }}</td>
+        <td bst-table-cell>{{ module.version }}</td>
+        <td bst-table-cell>{{ module.context }}</td>
+        <td bst-table-cell>{{ module.arch }}</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
```
Comment (johnpmitsch) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-module-streams.html]: If this is the same as the other PR, you can make this a shared template/partial

Comment (parthaa) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-module-streams.html]: good call

File: app/models/katello/module_stream.rb [hunk 2]
```diff
@@ -7,8 +7,16 @@ class ModuleStream < ApplicationRecord
...
 
-    CONTENT_TYPE = Pulp::ModuleStream::CONTENT_TYPE
+    scoped_search on: :name, complete_value: true
+    scoped_search on: :uuid, complete_value: true
+    scoped_search on: :stream, complete_value: true
+    scoped_search on: :version, complete_value: true
+    scoped_search on: :context, complete_value: true
+    scoped_search on: :arch, complete_value: true
+    scoped_search relation: :repositories, on: :name, rename: :repository, complete_value: true
+    scoped_search relation: :repositories, on: :id, rename: :repository_id, complete_value: true
```
Comment (jlsherrill) [inline app/models/katello/module_stream.rb]: does repo id really need to be searchable?

**Comment by parthaa** [comment]
Closing this .. Comments here are going to be addressed in https://github.com/Katello/katello/pull/7698

**Pull request closed without merging**